problems/secondSmallest.js

```diff
@@ -5,9 +5,22 @@
 * @param {number[]} arr - The input array of numbers
 * @returns {number} - Returns the second smallest number.
 */
+const secondSmallest = arr => {
+    if(nums.length < 2) return null
+      let firstSmallest = -1
```

Comment:
Staring with a number so small is going to make it hard for our function to work with arrrays that have values of 0 and greater. Instead we should start our numbers at something crazy big like Infinity to ensure that the values in the array will be smaller and trigger the updates.